main.go

```diff
@@ -46,6 +46,7 @@ func init() {
 	flag.StringVar(&address, "address", "localhost:8080", "Address of the package-registry service.")
 	// This flag is experimental and might be removed in the future or renamed
 	flag.BoolVar(&dryRun, "dry-run", false, "Runs a dry-run of the registry without starting the web service (experimental)")
+	flag.BoolVar(&util.PackageValidationDisabled, "disable-package-validation", false, "Disable package content validation")
```

Comment:
Quite a long name for a flag but no better suggestion.

Reply:
It's one of these feature flag that you should be discouraged to use :)